tests/Unit/Commands/GenerateEnumsCommandTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Kirschbaum\Paragon\Commands\GenerateEnumsCommand;
+
+it('generates enums', function () {
+    // Assemble.
+    setupStatusEnumTestCase($this->app);
+
+    // Act.
+    $this->artisan(GenerateEnumsCommand::class);
+
+    // Assert.
+    expect(resource_path(config('paragon.enums.paths.generated') . DIRECTORY_SEPARATOR . 'Status.ts'))->toBeFile();
```

Comment:
should we do some string matching checks here to ensure it was generated correctly?